opentelemetry-sdk/tests/metrics/integration_test/test_exponential_bucket_histogram.py

```diff
@@ -135,8 +135,6 @@ def test_synchronous_delta_temporality(self):
 
             results.append(reader.get_metrics_data())
 
-        provider.shutdown()
```

Comment:
I'm wondering if the original purpose of these shutdown calls was to actually `flush` the metric reader? @ocelotl

Reply:
It doesn't matter :shrug: this change actually removes an unnecessary warning.